src/test/java/co/rsk/federate/FederationProviderFromFederatorSupportTest.java

```diff
@@ -35,9 +35,6 @@ class FederationProviderFromFederatorSupportTest {
     private BridgeConstants bridgeConstants;
     private NetworkParameters testnetParams;
     private Instant creationTime;
-
-    private static final int STANDARD_MULTISIG_FEDERATION_FORMAT_VERSION = FederationFormatVersion.STANDARD_MULTISIG_FEDERATION.getFormatVersion();
-    private static final int NON_STANDARD_ERP_FEDERATION_FORMAT_VERSION = FederationFormatVersion.NON_STANDARD_ERP_FEDERATION.getFormatVersion();
```

Comment:
Shouldn't we update the existing tests to check for the federation format version?

Reply:
indeed. good catch, thanks!

Reply:
done!